lib/dealership.rb

```diff
@@ -0,0 +1,68 @@
+class Dealership
+  attr_accessor :inventory, :name, :location, :lot_value
+
+  def initialize(name, location)
+    @name = name
+    @location = location
+    @inventory = []
+    @lot_value = 0
+  end
+
+  def inventory_count
+    @inventory.length
+  end
+
+  def add_car(car)
+    @inventory << car
+  end
+
+  def has_inventory?
+    if inventory_count == 0
+      false
+    else
+      true
+    end
+  end
+
+  def cars_by_make(string)
```

Comment:
Make sure your naming is descriptive. Your test is passing in a string but what is that string? Is it the make? So I would replace the word `string` with `make` to make it more readable.